components/Navbar.js

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+
+const Navbar = () => {
+  return (
+    <nav>
+        <div className="logo">
+            <h1>Kino On Mars</h1>
+        </div>
+        <div className="links">
+        <Link href="/"><a>Home</a></Link>
+        <a>Movies</a>
+        <a>Tickets</a>
+        <a>About</a>
+        </div>
+    </nav>
+  );
+}
+
+export default Navbar
```

Comment:
Inside Navbar the links are "hardcoded". An option could be a variable or file that contains/exports an array of objects containing relevant information. Then loop over the array to create the menu. This could provide more flexibility if we (or product owner) need to add, remove or delete elements from the menu.

Reply:
I agree, it makes things modular, which is nice! Though maybe this is something we can look into after basic functionality is in place!